lib/google_search_data_viewer/keyword/google_search_parser.ex

```diff
@@ -0,0 +1,61 @@
+defmodule GoogleSearchDataViewer.Keyword.GoogleSearchParser do
+  @css_search_selectors %{
+    top_adwords: "#tads > .uEierd a.sVXRqc[href]",
+    top_non_adwords: ".MhgNwc a[href]",
+    non_adwords: ".yuRUbf a[href]",
+    bottom_adwords: "#bottomads .uEierd a.sVXRqc[href]"
+  }
+
+  def get_url_data_from_html(html) do
+    {_, parsed_html} = Floki.parse_document(html)
+
+    url_data =
+      []
+      |> prepend_top_adwords(parsed_html)
+      |> prepend_top_non_adwords(parsed_html)
+      |> prepend_non_adwords(parsed_html)
+      |> prepend_bottom_adwords(parsed_html)
+
+    url_data
+  end
+
+  defp prepend_top_adwords(url_data, html) do
```

Comment:
I think we can avoid using `data` word by something more specific.
```suggestion
  defp prepend_top_adwords(url_stats, html) do
```

Reply:
I think it depends on if it holds the definition of `Data` or `Statistics`, ie stats. As I have already collected the URL and appended the fields describing the URL I guess it now more closely resembles Statistics at this point so I will rename it to stats

Reply:
I will therefor also rename the function to `get_url_stats_from_html`